sdk/liftrqumulo/Azure.ResourceManager.LiftrQumulo/api/Azure.ResourceManager.LiftrQumulo.netstandard2.0.cs

```diff
@@ -0,0 +1,480 @@
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.LiftrQumulo.JobDefinitionResource>> UpdateAsync(Azure.ResourceManager.LiftrQumulo.Models.JobDefinitionPatch patch, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+    public partial class JobRunCollection : Azure.ResourceManager.ArmCollection, System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.LiftrQumulo.JobRunResource>, System.Collections.Generic.IEnumerable<Azure.ResourceManager.LiftrQumulo.JobRunResource>, System.Collections.IEnumerable
+    {
+        protected JobRunCollection() { }
+        public virtual Azure.Response<bool> Exists(string jobRunName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<bool>> ExistsAsync(string jobRunName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.LiftrQumulo.JobRunResource> Get(string jobRunName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.ResourceManager.LiftrQumulo.JobRunResource> GetAll(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.LiftrQumulo.JobRunResource> GetAllAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.LiftrQumulo.JobRunResource>> GetAsync(string jobRunName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        System.Collections.Generic.IAsyncEnumerator<Azure.ResourceManager.LiftrQumulo.JobRunResource> System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.LiftrQumulo.JobRunResource>.GetAsyncEnumerator(System.Threading.CancellationToken cancellationToken) { throw null; }
+        System.Collections.Generic.IEnumerator<Azure.ResourceManager.LiftrQumulo.JobRunResource> System.Collections.Generic.IEnumerable<Azure.ResourceManager.LiftrQumulo.JobRunResource>.GetEnumerator() { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+    }
+    public partial class JobRunData : Azure.ResourceManager.Models.ResourceData
+    {
+        public JobRunData() { }
+        public string AgentName { get { throw null; } }
+        public string AgentResourceId { get { throw null; } }
```

Comment:
Is this a `arm-id`? If yes please correct it by following https://github.com/Azure/autorest.csharp/blob/feature/v3/docs/mgmt/polishing.md#change-the-format-of-a-property